index.html

```diff
@@ -14,22 +14,30 @@
 </head>
 
 <body>
-    <header class="flex">
-        <div class="nav-bar flex">
+    <header class="">
+        <div class="nav-bar">
```

Comment:
- [ ] OPTIONAL: I strongly recommend making the background more blurry, To do that you can use `backdrop-filter` in css.